### PR TITLE
Ac rework

### PR DESCRIPTION
**Pull request by Moeriki**

### CODE REVIEW

File: app/src/vite-types/components.d.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ declare module '@vue/runtime-core' {
     TransitionChild: typeof import('@headlessui/vue')['TransitionChild']
     TransitionRoot: typeof import('@headlessui/vue')['TransitionRoot']
     UiBreadcrumb: typeof import('./../ui/breadcrumb/ui-breadcrumb.vue')['default']
-    UiBreadcrumbLink: typeof import('./../ui/breadcrumb/ui-breadcrumb-link.vue')['default']
+    UiBreadcrumbItem: typeof import('./../ui/breadcrumb/ui-breadcrumb-item.vue')['default']
```
Comment (SiteSeerBE) [inline app/src/vite-types/components.d.ts:60]: `<UiBreadcrumbLink to="">` has not been replaced by `<UiBreadcrumbItem linkTo="">` in components using it.

**Review by SiteSeerBE — CHANGES_REQUESTED** [review]

File: app/src/vite-types/components.d.ts (hunk 1 above)
Comment (Moeriki) [reply app/src/vite-types/components.d.ts:60]: Fixed.

Comment (Moeriki) [reply app/src/vite-types/components.d.ts:60]: @SiteSeerBE Ok?

Comment (SiteSeerBE) [reply app/src/vite-types/components.d.ts:60]: OK

**Pull request merged**